test/spec/services/angular-apimock.js

```diff
@@ -92,9 +92,10 @@ describe('Service: apiMock', function () {
 			$httpBackend.expect(defaultExpectMethod, defaultExpectPath).respond(404);
 
 			$http(defaultRequest)
-				.success(function () {
-					fail(); // Todo: How to fail the test if this happens?
-					failCb && failCb();
+				.success(/* istanbul ignore next - it is expecting a failure, so should fail tests if happen */
```

Comment:
Agree and understand it should be apart from this
Will revert and will let you know when ready.